tensorflow_gan/python/losses/losses_impl.py

```diff
@@ -58,6 +58,8 @@
     'mutual_information_penalty',
     'combine_adversarial_loss',
     'cycle_consistency_loss',
+    'ragan_generator_loss',
```

Comment:
what do you think about the names "relativistic_generator_loss" instead? I think it's more descriptive.